### DISCUSSION TITLE
Droppable Gear: steal protection

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/Prophets-of-the-Labyrinth#352] [Ideas]
Upgrades: Blasting

**Comment by doubledh** [comment]
how much? up to a cap, or all of it?

I'm thinking Hermit crabs take other shells, for inspiration

**Reply by ntseng** [reply]
This is planned as the Wind Defensive gear Siphon Enchantment. Cooldown was a great design lever for this kind of mechanic, allowing players to steal the whole protection value, but not do it every turn.
